src/AbstractPluginManager.php

```diff
@@ -167,7 +173,9 @@ public function get($name, ?array $options = null)
             $this->setFactory($name, Factory\InvokableFactory::class);
         }
 
-        $instance = empty($options) ? parent::get($name) : $this->build($name, $options);
+        $options = $options ?? [];
```

Comment:
Instead of creating a new array (expensive), let's check if it is `null`, and call `build()` if it is **not** null? I would also expect an explicit call to `build()` even if an empty array is given...

Reply:
Keeping `empty()` could also be a sufficient solution for now

Reply:
I've added `!$options` - that would avoid calling `empty` and does not require a new array.
Would that be sufficient?